src/style.css

```diff
@@ -1 +1,43 @@
 /* add styles here */
+.html {
```

Comment:
The selector `.html` should be `html` without the dot, as `html` is an HTML element, not a class.